tools/prove_wrapper

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+OUTPUT=$(mktemp)
+
+echo "Running prove with TAP output check ..."
+
+prove "$@" 2>&1 | tee "$OUTPUT"
+STATUS=${PIPESTATUS[0]}
+
+if [ "$STATUS" -ne 0 ]; then
+    echo "not ok - prove failed"
+    exit "$STATUS"
+fi
+
+sed -E '
```

Comment:
Does this already cover skipped tests as well? If not than that's probably needed.

Reply:
skipped test is considered as successful, so yes